### PR TITLE
OPSEXP-2407: add when condition in post_tasks for consistency

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-2395

aims at "fixing" #716

### CODE REVIEW

File: playbooks/acs.yml [hunk 1]
```diff
@@ -38,11 +38,14 @@
...
   roles:
     - role: "../roles/postgres"
-      when: repo_db_url == "" or sync_db_url == ""
+      when: test_env
   post_tasks:
     - name: Make sure PostgreSQL is running
+      when: test_env
       ansible.builtin.service:
         name: "{{ postgresql_service }}"
         state: started
```
Comment (gionn) [inline playbooks/acs.yml:51]: we could also say that if you have external database then you should not have any host in the database group

a better fix would be to adopt the same external_something approach we are using for other components so become more natural having to deal with inventory to define the infrastructure, but for now looks fine to me telling to drop the host from the database group

File: playbooks/acs.yml [hunk 2]
```diff
@@ -268,6 +271,7 @@
       when: acs.edition == "Enterprise"
   post_tasks:
     - name: Update installation status file with Trouter
+      when: acs.edition == "Enterprise"
```
Comment (gionn) [inline playbooks/acs.yml:274]: were they leaving an empty file with community or something?

File: playbooks/acs.yml (hunk 1 above)
Comment (alxgomz) [reply playbooks/acs.yml:51]: Couldn't agree more on aligning with other external services approach, though that's an heavier refactoring we can only plan for a major bump.
I have created OPSEXP-2408 for that (I was sure we already had a ticket for that but couldn't find it).
Are you suggesting we should not fix it and simply advertise one should make sure no database host remain in the inventoryif the repo_db_url are specified?

File: playbooks/acs.yml (hunk 2 above)
Comment (alxgomz) [reply playbooks/acs.yml:274]: No actually it was setting the trouter version set in `group_vars/all.yml` as it is a root element and the community extra vars file is not nulling it.

File: playbooks/acs.yml (hunk 1 above)
Comment (gionn) [reply playbooks/acs.yml:51]: yes we should not fix it, we are adding additional conditional logic here which is already implemented by `hosts` and the desired behaviour would be obtained by not defining the host in the database group, so for me it's more a misconfiguration than an actual bug.

Comment (gionn) [reply playbooks/acs.yml:51]: ⬆️

**Review by gionn — CHANGES_REQUESTED** [review]

**Comment by alxgomz** [comment]
superseeded by #721

**Pull request closed without merging**